protocol/app/prepare/full_node_prepare_proposal_test.go

```diff
@@ -1,220 +1,67 @@
 package prepare_test
 
 import (
+	"bytes"
 	"testing"
+	"time"
 
+	gometrics "github.com/armon/go-metrics"
 	abci "github.com/cometbft/cometbft/abci/types"
+	"github.com/cometbft/cometbft/libs/log"
 	sdktypes "github.com/cosmos/cosmos-sdk/types"
-	"github.com/dydxprotocol/v4-chain/protocol/app/prepare"
-	"github.com/dydxprotocol/v4-chain/protocol/mocks"
-	keepertest "github.com/dydxprotocol/v4-chain/protocol/testutil/keeper"
-	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
-	perpetualtypes "github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
-	pricestypes "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
-	"github.com/stretchr/testify/mock"
+	"github.com/dydxprotocol/v4-chain/protocol/app/flags"
+	testApp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
 	"github.com/stretchr/testify/require"
 )
 
 // TestFullNodePrepareProposalHandler test that the full-node PrepareProposal handler always returns
 // an empty result.
 func TestFullNodePrepareProposalHandler(t *testing.T) {
-	tests := map[string]struct {
-		txs      [][]byte
-		maxBytes int64
+	t.Cleanup(gometrics.Shutdown)
 
-		pricesResp    *pricestypes.MsgUpdateMarketPrices
-		pricesEncoder sdktypes.TxEncoder
+	conf := gometrics.DefaultConfig("testService")
+	sink := gometrics.NewInmemSink(time.Hour, time.Hour)
+	_, err := gometrics.NewGlobal(conf, sink)
+	require.NoError(t, err)
 
-		fundingResp    *perpetualtypes.MsgAddPremiumVotes
-		fundingEncoder sdktypes.TxEncoder
-
-		clobResp    *clobtypes.MsgProposedOperations
-		clobEncoder sdktypes.TxEncoder
-	}{
-		"Error: newPrepareProposalTransactions fails": {
```

Comment:
to confirm, are we removing these because these cases are irrelevant? (i.e. we don't care about the content of `RequestPrepareProposal` because we simply care that this method to be NOT called)

Reply:
That is right.